SU2_CFD/src/solver_direct_mean_inc.cpp

```diff
@@ -4226,10 +4226,11 @@ void CIncEulerSolver::SetPrimitive_Limiter(CGeometry *geometry, CConfig *config)
 
       for (iVar = 0; iVar < nPrimVarGrad; iVar++) {
         
-        if (config->GetKind_SlopeLimit_Flow() == VENKATAKRISHNAN_WANG)
-          eps2 = LimK *( GlobalMaxPrimitive[iVar] - GlobalMinPrimitive[iVar] );
-        else
-          eps2 = eps1*eps1*eps1;
+        if (config->GetKind_SlopeLimit_Flow() == VENKATAKRISHNAN_WANG) {
+          eps1 = 0.03 * (GlobalMaxPrimitive[iVar] - GlobalMinPrimitive[iVar]);
```

Comment:
Looks like the VEKATAKRISHNAN_WANG limiter coefficient is hard-coded. Any reason to make this variable, or does the tuning within this new limiter take care of any required scaling?

Reply:
Thanks!

There is a good reason for that... my frustration using the original Venkat's limiter. The freedom to choose parameters is a double-edged sword... you have control but you can affect the results. With the original method (2 parameters)... I was able to obtain any result! 

For that reason, in this new implementation I decided to hardcode the recommended value for the parameter. So... users will have consisten solutions.

By the way, I have some results that point out to the fact that this version is much better than the original Venkat's limiter (convergence and accuracy)... maybe in the near future we should replace the original one by this one.

Best,
Vivaan

Reply:
Dear all,

Thanks for the implementation @VivaanKhatri 

However, the eps value in VEKATAKRISHNAN_WANG can be view as percentage of the maximum range of the flow field, oscillations exceeding this value are limited.  In my opinion, this value should not be hard-coded and we can use the limiter coefficient from the config file. According to Wang's original "A fast nested multi-grid viscous flow solver", the value of 0.05 is a good starting candidate. 

Best,

Eduardo

Reply:
I have added the option LIMITER_COEFF to the VENKATAKRISHNAN_WANG limiter.

Best,
Vivaan